CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ It is based on Tidepool Blip 1.27.
 - YLP-1345 Correct blip pdf export to display japanese chars
 - YLP-1383 Rework app routing
 - YLP-1447 Skip flaky test
+- YLP-1506 Skip flaky test
```

Comment:
can't we use the same jira ticket number ?

Reply:
No